### ISSUE TITLE
Exit button for if bugs

### ISSUE THREAD

**Issue by janat08** [qualialabs/reval#16]
Using blaze-components the page goes through the UI, so if exit button is suppose to be in that page pane then it doesn't render, and obviusly you can't use the page right. It would be cool if the state would save between exits.

**Comment by veered** [comment]
Could you post a screen shot?

**Comment by janat08** [comment]
![image](https://user-images.githubusercontent.com/11337345/27646584-d0aee38a-5c4a-11e7-8d2e-7841d276078f.png)

**Comment by veered** [comment]
Are you talking about the `</>`?

**Comment by janat08** [comment]
Yes. That deserves it's on space, and look at the page running through edge of the screen and no scrollbar.

**Comment by janat08** [comment]
The clear all button is missing now.

**Comment by veered** [comment]
I didn't push any changes recently. Screenshot?

**Comment by janat08** [comment]
![image](https://user-images.githubusercontent.com/11337345/27765365-eac8627c-5ed0-11e7-9f0b-e1bbdcb8ec91.png)
https://github.com/qualialabs/reval/issues/16#issuecomment-312454099

**Comment by janat08** [comment]
full screen in browser on laptop also has it gone. i feel ashamed.

**Comment by veered** [comment]
Are you sure you're using the newest version of Reval?

Also, I see you're using SUI :) The guy who makes it is our head of design! In fact, he's the one that added the "Clear All" button haha

**Comment by janat08** [comment]
Well I had clear all in the 1st screenshot of topic.

**Comment by janat08** [comment]
I imagine I removed the template, for I use Blaze Components so templates can totally intermingle if I returned null for template option, or applied some weird js on your SUI.
